gammapy/data/observations.py

```diff
@@ -346,51 +346,57 @@ def check(self, checks="all"):
         checker = ObservationChecker(self)
         return checker.run(checks=checks)
 
-    def peek(self, figsize=(12, 10)):
+    def peek(self, figsize=(15, 10)):
         """Quick-look plots in a few panels.
 
         Parameters
         ----------
         figsize : tuple
             Figure size
         """
-        n_irfs = len(self.available_hdus)
+
+        plottable_hds = ["events", "aeff", "psf", "edisp", "bkg", "rad_max"]
```

Comment:
Instead of repeating the IRF names here it would be better to just use `self.available_hdus` and remove "gti" from it, which is the only one which is not plottable.

Reply:
That was my initial implementation, but as @registerrier pointed out, that leaves the risk of adding additional sub plots if hdus are added in future

Reply:
It's close to 50:50 I guess, because one could also argue that if additional HDUs are supported they should be plotted as well, if available. But OK to keep from my side for now...